package.json

```diff
@@ -11,9 +11,9 @@
         "clean": "ionic-app-scripts clean",
         "build": "ionic-app-scripts build",
         "dist": "npm run build && npm run dist:linux",
-        "dist:darwin": "electron-packager . --ignore='/^(?!www).*/' --platform=darwin --arch=x64 --asar --out=dist/ --overwrite",
+        "dist:darwin": "electron-packager . --ignore=\"/^(?!www).*/\" --platform=darwin --arch=x64 --asar --out=dist/ --overwrite",
```

Comment:
Could you make sure that this modification is also needed under Darwin/Mac?

I don't think so...